rllib/optimizers/aso_aggregator.py

```diff
@@ -113,7 +113,7 @@ def iter_train_batches(self, max_yield=999):
         """
 
         for ev, sample_batch in self._augment_with_replay(
-                self.sample_tasks.completed_prefetch(
+                self.sample_tasks.completed(
```

Comment:
Historically, changes to this file have been very performance sensitive. Can you run the YAML here https://github.com/ray-project/rl-experiments/tree/master/atari-impala

before/after the change to ensure there isn't any significant regression?

In particular I would verify Breakout reaches the same reward / similar timestep throughput after say 1 hr as before on a p3.16xl instance, you can probably ignore the others.